src/Castle.Windsor.Tests/Categories/UnitTestAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2004-2017 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace CastleTests.Categories
+{
+	using NUnit.Framework;
+
+	public class UnitTestAttribute : CategoryAttribute
+	{
+		public UnitTestAttribute() : base("Unit")
```

Comment:
My design suggestion in #174 of only a single integration category and using `Explicit` for integration tests was for simplicity and familiarity for users, but importantly so that integration tests don't automatically get run by ReSharper or the VS test runner. I'd also much prefer not to have to remember to put an extra `UnitTest` attribute on classes, especially that `TestFixture` is no longer required in NUnit 2.5 so we can start omitting that attribute.

Can you explain the need for the `ManualTest` attribute, isn't that what explicit is for, which is why they are already marked as such?